composer.json

```diff
@@ -16,7 +16,8 @@
     "psr-4": {
       "PublishPress\\Checklists\\Core\\": "core/",
       "PublishPress\\Checklists\\Permalinks\\": "modules/permalinks/lib",
-      "PublishPress\\Checklists\\Yoastseo\\": "modules/yoastseo/lib"
+      "PublishPress\\Checklists\\Yoastseo\\": "modules/yoastseo/lib",
+      "PublishPress\\Checklists\\Customs\\": "modules/customs/lib"
     }
```

Comment:
The name `customs` only makes sense if you are moving all the custom items feature into that module.

Reply:
Okay, i'll change the module name but the reason i named it custom is not because of the feature but to have a module that contain all the future features that can't be map to post attribute unlike like editor, title, excerpt etc that are directly added in core requirement

Reply:
So, should i proceed with the module been limited to the function which is role approval ? @andergmartins

Reply:
Yes please. Limit it to only this feature for now. Thanks. More info here: https://github.com/publishpress/PublishPress-Checklists/pull/183#issuecomment-668779023